content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -0,0 +1,409 @@
+
+```bash
+dvc exp run
+```
+
+You'll see output similar to this in your terminal while the training process is
+going on.
+
+```dvc
+Checkpoint experiment iteration 'd50c724'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration '29491a9'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'b3de55f'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'c4a46af'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'daf204c'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration 'bdb975c'.
+Updating lock file 'dvc.lock'
+Checkpoint experiment iteration '77dc46c'.
+Updating lock file 'dvc.lock'
+```
+
+After a few epochs have completed, stop terminate the training process with
+`Ctrl + C`. Now it's time to take a look at the metrics we're working with.
+
+_If you don't have a number of training epochs defined and you don't terminate
+the process, the experiment will run for 100 epochs._
+
+## Viewing checkpoints
+
+You can see a table of your experiments and checkpoints in the terminal by
+running:
+
+```bash
+dvc exp show
+```
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created  ┃ step ┃    loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -        │    7 │  0.3096 │ 0.9092 │ 473987 │ 0.0001 │ 0            │
+│ main                  │ 04:49 PM │    - │       - │      - │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-4de2a         │ 05:03 PM │    7 │  0.3096 │ 0.9092 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ d50c724           │ 05:03 PM │    6 │ 0.33246 │ 0.9044 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 29491a9           │ 05:02 PM │    5 │ 0.36601 │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b3de55f           │ 05:02 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ c4a46af           │ 05:02 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ daf204c           │ 05:02 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ bdb975c           │ 05:02 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ 77dc46c           │ 05:01 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+```
+
+## Starting from an existing checkpoint
+
+Since you have all of these different checkpoints, you might want to resume
+training from a particular one. For example, maybe your accuracy started
+decreasing at a certain checkpoint and you want to make some changes to fix
+that.
+
+First, we need to apply the checkpoint we want to begin our new experiment from.
+To do that, run the following command:
+
+```bash
+dvc exp apply b3de55f
+```
+
+where _b3de55f_ is the id of the checkpoint you want to reference.
+
+Next, we'll change the learning rate in the _params.yaml_ to `0.001`. Then we'll
```

Comment:
just an idea:
```
dvc exp run --set-param lr=0.001
``` 
might be an easier way (for user) of doing it.

Reply:
@dmpetrov @dberenbaum I tried running this command in few different ways and I get this error each time:

```
Traceback (most recent call last):
  File "/Users/milecia/Repos/checkpoints-tutorial/train.py", line 123, in <module>
    main()
  File "/Users/milecia/Repos/checkpoints-tutorial/train.py", line 110, in main
    train(model, x_batch, y_batch, params["lr"], params["weight_decay"])
  File "/Users/milecia/Repos/checkpoints-tutorial/train.py", line 45, in train
    optimizer = torch.optim.Adam(model.parameters(), lr=lr,
  File "/Users/milecia/Repos/checkpoints-tutorial/.venv/lib/python3.9/site-packages/torch/optim/adam.py", line 36, in __init__
    if not 0.0 <= lr:
TypeError: '<=' not supported between instances of 'float' and 'str'
```

Any thoughts or suggestions?